extensions/cornerstone/src/commandsModule.ts

```diff
@@ -2063,27 +2064,30 @@ function commandsModule({
         });
       }
     },
-    increaseBrushSize: () => {
+    _handleBrushSizeAction: (action: 'increase' | 'decrease') => {
       const toolGroupIds = toolGroupService.getToolGroupIds();
       if (!toolGroupIds?.length) {
         return;
       }
 
       for (const toolGroupId of toolGroupIds) {
         const brushSize = segmentationUtils.getBrushSizeForToolGroup(toolGroupId);
-        segmentationUtils.setBrushSizeForToolGroup(toolGroupId, brushSize + 3);
+
+        const newBrushSize =
+          action === 'increase' ? Math.min(brushSize + 3, 99.5) : Math.max(brushSize - 3, 0.5);
```

Comment:
One thing that bothers me is that this capping between >= 0.5 and <= 99.5 could differ what is specified in the button options. Which is fine in a sense because this logic could and should exist outside of there being a button/control. However, the button evaluator should also call `segmentationUtils.setBrushSizeForToolGroup` for the case where the capping is different.